localstripe/resources.py

```diff
@@ -2332,7 +2336,9 @@ def __init__(self, customer=None, metadata=None, items=None,
         self.trial_start = None
         self.trial_period_days = trial_period_days
         self.latest_invoice = None
-        self.start_date = int(time.time())
+        self.start_date = backdate_start_date \
+            if backdate_start_date is not None \
+            else int(time.time())
```

Comment:
Just for style consistency, could you change to:
```suggestion
        self.start_date = (backdate_start_date
                           if backdate_start_date is not None
                           else int(time.time()))
```
or
```suggestion
        self.start_date = backdate_start_date or int(time.time())
```
?

Reply:
Thanks for the comments, I addressed them in a new commit.